Storybook is broken on master / next branches

Hello! I tried pulling down this repo and running the storybook locally, but the build appears to be broken. 

This is when running `yarn` then `yarn storybook` in the root directory.

`yarn: 1.22.4`
`node: 14.5.0`
`macOS 10.15.5`

Log: 
```
yarn run v1.22.4
$ start-storybook -p 6006
info @storybook/react v5.3.19
info 
info => Loading presets
info => Loading presets
info => Loading custom manager config.
info => Loading config/preview file in "./.storybook".
info => Adding stories defined in ".storybook/main.js".
info => Using default Webpack setup.
info => Using base config because react-scripts is not installed.
webpack built bb9ea818b45bf65081e3 in 65119ms
✖ ｢wdm｣: Hash: bb9ea818b45bf65081e3
Version: webpack 4.43.0
Time: 65119ms
Built at: 07/17/2020 5:16:23 PM
                                          Asset      Size        Chunks                                Chunk Names
                                    iframe.html  2.87 KiB                [emitted]                     
            main.bb9ea818b45bf65081e3.bundle.js  3.88 MiB          main  [emitted] [immutable]  [big]  main
        main.bb9ea818b45bf65081e3.bundle.js.map  3.35 MiB          main  [emitted] [dev]               main
    runtime~main.bb9ea818b45bf65081e3.bundle.js  33.5 KiB  runtime~main  [emitted] [immutable]         runtime~main
runtime~main.bb9ea818b45bf65081e3.bundle.js.map  34.7 KiB  runtime~main  [emitted] [dev]               runtime~main
    vendors~main.bb9ea818b45bf65081e3.bundle.js  13.8 MiB  vendors~main  [emitted] [immutable]  [big]  vendors~main
vendors~main.bb9ea818b45bf65081e3.bundle.js.map  12.2 MiB  vendors~main  [emitted] [dev]               vendors~main
Entrypoint main [big] = runtime~main.bb9ea818b45bf65081e3.bundle.js runtime~main.bb9ea818b45bf65081e3.bundle.js.map vendors~main.bb9ea818b45bf65081e3.bundle.js vendors~main.bb9ea818b45bf65081e3.bundle.js.map main.bb9ea818b45bf65081e3.bundle.js main.bb9ea818b45bf65081e3.bundle.js.map
[0] multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/preview.js ./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=true 112 bytes {main} [built]
[./.storybook/generated-entry.js] 2.01 KiB {main} [built]
[./.storybook/preview.js] 563 bytes {main} [built]
[./node_modules/@storybook/addon-docs/blocks.js] 43 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js] 292 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js] 672 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-docs/dist/frameworks/react/extractProps.js] 2.21 KiB {vendors~main} [built]
[./node_modules/@storybook/addon-docs/dist/lib/docgen/index.js] 932 bytes {vendors~main} [built]
[./node_modules/@storybook/addon-knobs/dist/index.js] 6.3 KiB {vendors~main} [built]
[./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js] 149 bytes {vendors~main} [built]
[./node_modules/@storybook/client-api/dist/index.js] 3.82 KiB {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/common/polyfills.js] 120 bytes {vendors~main} [built]
[./node_modules/@storybook/core/dist/server/preview/globals.js] 93 bytes {vendors~main} [built]
[./node_modules/@storybook/react/dist/client/index.js] 1.34 KiB {vendors~main} [built]
[./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=true] (webpack)-hot-middleware/client.js?reload=true&quiet=true 7.68 KiB {vendors~main} [built]
    + 4516 hidden modules

WARNING in ./packages/slate-plugins/src/deserializers/deserialize-md/withDeserializeMd.ts 18:19-25
"export 'default' (imported as 'marked') was not found in 'marked'
 @ ./packages/slate-plugins/src/deserializers/deserialize-md/index.ts
 @ ./packages/slate-plugins/src/deserializers/index.ts
 @ ./packages/slate-plugins/src/index.ts
 @ ./stories/elements/editable-voids/editable-voids.stories.tsx
 @ ./stories sync ^\.\/(?:(?:|\/|(?:(?:(?!(?:|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(tsx|mdx))$
 @ ./.storybook/generated-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/preview.js ./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=true

ERROR in ./packages/slate-plugins/src/index.ts
Module not found: Error: Can't resolve '@udecode/slate-plugins-core' in '/Users/sallya/Developer/slate-plugins/packages/slate-plugins/src'
 @ ./packages/slate-plugins/src/index.ts 1:0-44 1:0-44
 @ ./stories/elements/editable-voids/editable-voids.stories.tsx
 @ ./stories sync ^\.\/(?:(?:|\/|(?:(?:(?!(?:|\/)\.).)*?)\/)(?!\.)(?=.)[^/]*?\.stories\.(tsx|mdx))$
 @ ./.storybook/generated-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/preview.js ./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=true
Child HtmlWebpackCompiler:
                          Asset      Size               Chunks  Chunk Names
    __child-HtmlWebpackPlugin_0  6.33 KiB  HtmlWebpackPlugin_0  HtmlWebpackPlugin_0
    Entrypoint HtmlWebpackPlugin_0 = __child-HtmlWebpackPlugin_0
    [./node_modules/html-webpack-plugin/lib/loader.js!./node_modules/@storybook/core/dist/server/templates/index.ejs] 2.13 KiB {HtmlWebpackPlugin_0} [built]

WARN Broken build, fix the error above.
WARN You may need to refresh the browser.

webpack building...
webpack built 8533373caac8ea8b6718 in 6640ms
✖ ｢wdm｣: Hash: 8533373caac8ea8b6718
Version: webpack 4.43.0
Time: 6640ms
Built at: 07/17/2020 5:16:33 PM
                                          Asset      Size        Chunks                                      Chunk Names
           bb9ea818b45bf65081e3.hot-update.json  35 bytes                [emitted] [immutable] [hmr]         
                                    iframe.html  2.87 KiB                [emitted]                           
            main.8533373caac8ea8b6718.bundle.js  3.88 MiB          main  [emitted] [immutable]        [big]  main
        main.8533373caac8ea8b6718.bundle.js.map  3.35 MiB          main  [emitted] [dev]                     main
    runtime~main.8533373caac8ea8b6718.bundle.js  33.5 KiB  runtime~main  [emitted] [immutable]               runtime~main
runtime~main.8533373caac8ea8b6718.bundle.js.map  34.7 KiB  runtime~main  [emitted] [dev]                     runtime~main
    vendors~main.8533373caac8ea8b6718.bundle.js  13.8 MiB  vendors~main  [emitted] [immutable]        [big]  vendors~main
vendors~main.8533373caac8ea8b6718.bundle.js.map  12.2 MiB  vendors~main  [emitted] [dev]                     vendors~main
Entrypoint main [big] = runtime~main.8533373caac8ea8b6718.bundle.js runtime~main.8533373caac8ea8b6718.bundle.js.map vendors~main.8533373caac8ea8b6718.bundle.js vendors~main.8533373caac8ea8b6718.bundle.js.map main.8533373caac8ea8b6718.bundle.js main.8533373caac8ea8b6718.bundle.js.map
[0] multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/preview.js ./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=true 112 bytes {main}
[./.storybook/generated-entry.js] 2.01 KiB {main} [built]
[./.storybook/preview.js] 563 bytes {main}
[./node_modules/@storybook/addon-docs/blocks.js] 43 bytes {vendors~main}
[./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js] 292 bytes {vendors~main}
[./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js] 672 bytes {vendors~main}
[./node_modules/@storybook/addon-docs/dist/frameworks/react/extractProps.js] 2.21 KiB {vendors~main}
[./node_modules/@storybook/addon-docs/dist/lib/docgen/index.js] 932 bytes {vendors~main}
[./node_modules/@storybook/addon-knobs/dist/index.js] 6.3 KiB {vendors~main}
[./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js] 149 bytes {vendors~main}
[./node_modules/@storybook/client-api/dist/index.js] 3.82 KiB {vendors~main}
[./node_modules/@storybook/core/dist/server/common/polyfills.js] 120 bytes {vendors~main}
[./node_modules/@storybook/core/dist/server/preview/globals.js] 93 bytes {vendors~main}
[./node_modules/@storybook/react/dist/client/index.js] 1.34 KiB {vendors~main}
[./node_modules/webpack-hot-middleware/client.js?reload=true&quiet=true] (webpack)-hot-middleware/client.js?reload=true&quiet=true 7.68 KiB {vendors~main}
    + 4516 hidden modules

WARNING in ./packages/slate-plugins/src/deserializers/deserialize-md/withDeserializeMd.ts 18:19-25
"export 'default' (imported as 'marked') was not found in 'marked'
 @ ./packages/slate-plugins/src/deserializers/deserialize-md/index.ts
 @ ./packages/slate-plugins/src/deserializers/index.ts
 @ ./packages/slate-plugins/src/index.ts
 @ ./stories/examples/playground.stories.tsx
 @ ./stories/examples sync nonrecursive ^\.\/(?:(?:playground\.stories\.tsx)$)$
 @ ./.storybook/generated-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/preview.js ./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=true

ERROR in ./packages/slate-plugins/src/index.ts
Module not found: Error: Can't resolve '@udecode/slate-plugins-core' in '/Users/sallya/Developer/slate-plugins/packages/slate-plugins/src'
 @ ./packages/slate-plugins/src/index.ts 1:0-44 1:0-44
 @ ./stories/examples/playground.stories.tsx
 @ ./stories/examples sync nonrecursive ^\.\/(?:(?:playground\.stories\.tsx)$)$
 @ ./.storybook/generated-entry.js
 @ multi ./node_modules/@storybook/core/dist/server/common/polyfills.js ./node_modules/@storybook/core/dist/server/preview/globals.js ./node_modules/@storybook/addon-docs/dist/frameworks/common/config.js ./node_modules/@storybook/addon-docs/dist/frameworks/react/config.js ./.storybook/preview.js ./node_modules/@storybook/addon-knobs/dist/preset/addDecorator.js ./.storybook/generated-entry.js (webpack)-hot-middleware/client.js?reload=true&quiet=true
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -279,8 +279,9 @@ are on the latest versions (yarn at least v1.3.2).
    _bonus_: use your own fork for this step
 3. `cd slate-plugins`
 4. `yarn`
-5. `yarn test`
-6. `yarn storybook`
+5. `yarn build`
+6. `yarn test` (optional)
+7. `yarn storybook`
 
 ### Working with Storybook
 

--- a/README.md
+++ b/README.md
@@ -192,9 +192,13 @@ Useful scripts include:
 
 > Installs package dependencies
 
+#### `yarn build`
+
+> Build the local packages.
+
 #### `yarn storybook`
 
-> Starts storybook dev
+> Starts storybook dev (after building).
 
 #### `yarn lint`
 

